src/main/jbake/content/blog/2017/04/07/cloudnativecon-summary.adoc

```diff
@@ -0,0 +1,48 @@
+= CloudNativeCon summary
+Juraci Paixão Kröhling
+2017-04-07
+:jbake-type: post
+:jbake-status: published
+:jbake-tags: blog, apm, opentracing, cloudnativecon
+
+CloudNativeCon happened last week in Berlin and Hawkular APM was represented there
+with a talk, link:https://www.youtube.com/watch?v=utCOfSz202o["Operational and Business Insights from Tracing Distributed Microservices"].
+Here's a short summary of the event, with a focus on OpenTracing.
+
+This event was co-located with KubeCon and pretty much everything floated around Kubernetes.
+There were quite a few tracing vendors with booths and/or in the audience:
+Instana, DataDog, Dynatrace, Sysdig, CA Technologies, among others. It was a great
+opportunity to talk to them, demo Hawkular APM to them, watch their demos, exchange ideas
+and invite some of them to the OpenTracing community.
```

Comment:
Understand - the discussions would be specific to the individual cases, but just meant that the blog could be interpreted that way - so better not to include 'some of' to sound more inclusive.